mvpa2/tests/test_searchlight_hyperalignment.py

```diff
@@ -208,7 +208,7 @@ def test_searchlight_hyperalignment(self):
         for ds in dss[1:]:
             assert_false(np.all(ds_orig.samples == ds.samples))
         # testing checks
-        slhyp = SearchlightHyperalignment(ref_ds=1, exclude_from_model=[1])
+        slhyp = SearchlightHyperalignment(ref_ds=2, exclude_from_model=[1])
```

Comment:
I think an explicit test is a better option. The model (including feature selection) should be the same, and results of other datasets shouldn't change. However, results of the excluded datasets should change as we're using different .samples for them, right?